guides/common/modules/proc_exporting-templates.adoc

```diff
@@ -15,24 +15,31 @@ The {ProjectWebUI} displays the status of the export.
 The status is not persistent; if you leave the status page, you cannot return to it.
 
 .CLI procedure
-. Clone a local copy of your Git repository:
+. Clone a local copy of your Git repository to a subdirectory of `/var/lib/foreman`:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-$ git clone https://github.com/theforeman/community-templates __/custom/templates__
+$ git clone https://github.com/theforeman/community-templates /var/lib/foreman/community-templates
 ----
-. Change the owner of your local directory to the `foreman` user, and change the SELinux context with the following commands:
++
+[NOTE]
+====
+You can clone your repository to a custom directory outside `/var/lib/foreman`, but you have to ensure that the `Foreman` service can read its contents.
```

Comment:
```suggestion
You can clone your repository to a custom directory outside `/var/lib/foreman`.
Ensure that the `Foreman` service can read its contents.
```
Minimalism.

Reply:
I'd prefer to keep this as one sentence as it is a direct warning that is the user wishes  to clone to a custom directory, they must ensure that these conditions are met.

Reply:
You are already on `Note` so the importance remains the same as you want. But, final call is yours :+1: